imagepicker/src/main/java/com/esafirm/imagepicker/features/ImagePickerConfig.java

```diff
@@ -77,6 +79,24 @@ public void setSelectedImages(ArrayList<Image> selectedImages) {
         this.selectedImages = selectedImages;
     }
 
+    public ArrayList<File> getExcludedImages() {
+        return excludedImages;
+    }
+
+    public void setExcludedImages(ArrayList<Image> excludedImages) {
```

Comment:
I think this could be simpler

```java
if(excludeImages != null && !excludedImages.isEmpty()) {
    for (Image image : excludedImages) {
        this.excludedImages.add(new File(image.getPath());
    }
} 
```

Reply:
But then you can't set it to `null`.

Reply:
Any reason why do you want to set it to `null` ? 

We use a new `ImagePickerConfig` every time we start the image picker 

https://github.com/esafirm/android-image-picker/blob/master/imagepicker/src/main/java/com/esafirm/imagepicker/features/ImagePicker.java#L67